dev_guide/expose_service/expose_internal_ip_nodeport.adoc

```diff
@@ -78,16 +78,16 @@ spec:
     name: mysql
 ----
 
-. Execute the following command to xref:../../dev_guide/application_lifecycle/new_app.adoc#dev-guide-new-app[create the service]:
+. Execute the following command to create the service:
```

Comment:
To paraphrase the IBM Style Guide: Use **execute** only if a simpler, less technical term, such as "run" or a more specific term, such as "run" or "start" is inappropriate.

Reply:
Thanks! Submitted #20901 to update this.